pkg/coremodel/pluginmeta/coremodel.cue

```diff
@@ -15,9 +15,7 @@ seqs: [
 				// grafana.com, then the plugin id has to follow the naming
 				// conventions.
 				id: string & strings.MinRunes(1)
-				if !builtIn {
-						id: =~"^[0-9a-z]+\\-([0-9a-z]+\\-)?(app|panel|datasource)$"
-				}
+				id: =~"^([0-9a-z]+\\-([0-9a-z]+\\-)?(app|panel|datasource))|(alertGroups|alertlist|annolist|barchart|bargauge|candlestick|canvas|dashlist|debug|gauge|geomap|gettingstarted|graph|heatmap|heatmap-old|histogram|icon|live|logs|news|nodeGraph|piechart|pluginlist|stat|state-timeline|status-history|table|table-old|text|timeseries|traces|welcome|xychart|alertmanager|cloudwatch|dashboard|elasticsearch|grafana|grafana-azure-monitor-datasource|graphite|influxdb|jaeger|loki|mixed|mssql|mysql|opentsdb|postgres|prometheus|stackdriver|tempo|testdata|zipkin)$"
```

Comment:
OK, so i can totally see how this hacks in something that can at least lets us establish a floor where core plugins successfully validate. And it's a clear improvement over `builtIn` because we've gotten rid of conditionals, haven't sullied the schema with an unnecessary field, and don't have to do anything special with plugin publishing validation on marketplace.

But it also amounts to a logical loop:

* pluginmeta coremodel depends on this list of the core plugins
* core plugin code generator depends on the pluginmeta coremodel

The effect is that core plugin authors have to just know to update this list in order to create a new plugin. Also, because the list is closed, it _may_ create a possibility that older versions of Grafana reject newer version `plugin.json`s if they come from newer core plugins...though it's hard to see how that would happen, because the plugin list is compiled in.

So, i can see this as a temporary measure, but no more than that. IMO we can't consider this all baked until we've done something that does not involve a static list of the current set of core plugins.